server/controllers/apply/applicationsController.ts

```diff
@@ -78,4 +79,16 @@ export default class ApplicationsController {
       )
     }
   }
+
+  submit(): RequestHandler {
+    return async (req: Request, res: Response) => {
+      const application = await this.applicationService.findApplication(req.user.token, req.params.id)
+      application.document = getResponses(application)
+
+      await this.applicationService.saveToApi(application, req)
```

Comment:
Do we need to save to the API and submit here?

Reply:
We probably don't have to... 
But we only submit the translated document.
By saving it we save the application with the translated text so the API then has the final version of the full Application object. 
But the API must be doing some work to attatch the translated text to the application object anyway